terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -115,14 +120,17 @@ def mutate_and_get_payload(cls, root, info, create_soil_data=True, **kwargs):
         log = cls.get_logger()
         user = info.context.user
 
+        if not check_site_permission(user, "create", Context()):
+            cls.not_allowed_create(Site)
```

Comment:
question: below on line 134, we use `cls.not_allowed(MutationTypes.CREATE)`, and here we use `cls.not_allowed_create(Site)`, is there a difference?

Reply:
There's a slight difference in how the error codes are reported. This case seemed like a simple fix, but generally our error code handling isn't very consistent / wasn't consistent before this PR and it felt out of scope to make across the board standardization.

Reply:
Did you mean "make a ticket", or do you mean I should begin work on that immediately?

Reply:
Sorry, make a placeholder ticket for later work.

Reply:
Done - https://github.com/techmatters/terraso-backend/issues/1264